src/lib/fetch.js

```diff
@@ -47,7 +47,9 @@ function fetchWithAuth (method) {
     function errorHandler (error) {
       const resp = error.response
       checkErrors(resp)
-      throw new Error(error)
+      const newError = new Error(error)
+      newError.originalError = error
+      throw newError
```

Comment:
can we just `throw error?`

Reply:
I think it is done this way so that the stack trace makes more sense (points to this file, rather than something deep in axios)